src/components/KTruncate/KTruncate.vue

```diff
@@ -43,18 +44,20 @@
         >
           <ChevronUpIcon
             class="collapse-trigger"
-            :color="KUI_COLOR_TEXT_PRIMARY"
             role="button"
             tabindex="0"
             @click.stop="handleToggleClick"
+            @enter="handleToggleClick"
+            @keydown.space.prevent
+            @keyup.space="handleToggleClick"
```

Comment:
Couple questions:

- Shouldn't this be keydown.enter? 
- why are we binging space to keyup and then preventing default on keydown?

Reply:
- If I click enter or space on the expand button, it works.
- Only the space key works on the collapse button to close.
- The focus-visible state of the collapse button looks... odd
 
![image](https://github.com/Kong/kongponents/assets/2229946/3145a3c1-1a30-4354-9a78-9675e7f3757b)

Reply:
- Yes, my mistake - corrected
- Preventing default on space keydown so that it doesn't scroll to the element when pressing enter. Binding to keyup so that pressing space triggers the event

Reply:
For the expand button, it's not 100% obvious the count e.g. `+6` is a clickable button. Should it have a background color and border radius similar to the chevron icon?

Reply:
Design approved it the way it is -- we can change it later when design comes up with Figmas for this component (right now we don't have any)

Reply:
> * If I click enter or space on the expand button, it works.
> * Only the space key works on the collapse button to close.
> * The focus-visible state of the collapse button looks... odd
> 
> ![image](https://private-user-images.githubusercontent.com/2229946/305449926-3145a3c1-1a30-4354-9a78-9675e7f3757b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwOTU5MDMsIm5iZiI6MTcwODA5NTYwMywicGF0aCI6Ii8yMjI5OTQ2LzMwNTQ0OTkyNi0zMTQ1YTNjMS0xYTMwLTQzNTQtOWE3OC05Njc1ZTdmMzc1N2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMTZUMTUwMDAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGYyZWEwYTNiMTk4YzcwNDAzZTAyMzg3ZWE3NTQ2ZjViZjg1NzRiMWNkZGE5ZWQ3YTQwMjIyOWY2OTMwN2Y5YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.kOMy0VEJRUTPDZtOhjkfP0NuOwJ4Qs-PsvLbGyZ54hY)

Updated th component so all buttons work on click, enter and space. As for `focus-visible` styling - again, maybe let's correct that if we need when/if design wants to change it?

Reply:
I don't think changing the default behavior of buttons in the browser is desired (for this, and the other PRs)